homework3/test_hw_3/test_task04.py

```diff
@@ -0,0 +1,11 @@
+from homework3.task04.task04 import is_armstrong
+
+
+def test_is_armstrong_num_153():
+    """Testing that number 153 is Armstrong num"""
+    assert is_armstrong(153)
+
+
+def test_is_armstrong_num_10():
```

Comment:
test name should describe case you test inside. this name only tells about incoming data.
think about "test_number_is_not_armstrong".

also rename upper test case